services/viva-ms/lambdas/syncWorkflowAndSetStatus.js

```diff
@@ -7,99 +7,147 @@ import * as dynamoDb from '../../../libs/dynamoDb';
 import { getStatusByType } from '../../../libs/caseStatuses';
 import vivaAdapter from '../helpers/vivaAdapterRequestClient';
 
-const CASE_WORKFLOW_PATH = 'details.workflow';
-
 export async function main(event) {
   const personalNumber = event.detail.user.personalNumber;
-  const PK = `USER#${personalNumber}`;
 
-  const [getAllUserCasesError, allUserCases] = await to(getAllUserCases(PK));
-  if (getAllUserCasesError) {
-    return console.error('(Viva-ms) DynamoDB query failed', getAllUserCasesError);
+  const [getCasesError, userCases] = await to(
+    getCasesWithStatusSumbittedOrProcessing(personalNumber)
+  );
+  if (getCasesError) {
+    throw ('(Viva-ms) getCasesWithStatusSumbittedOrProcessing', getCasesError);
```

Comment:
This will not work as intended, as it'll just throw the value of `getCasesError`.
The `(Viva-ms) getCasesWithStatusSumbittedOrProcessing` part will **not** follow along in the error message.

For reference, try this in your browser console;

```js
const myError = "This is an error string"

try {
	throw ('I want this to log', myError)
} catch (e) {
	console.error(e)
}
```

All you'll get is the string `This is an error string`

Reply:
instead of throwing this error you could do something in the lines of:
```
if (getCasesError) {
   console.error('(Viva-ms) getCasesWithStatusSumbittedOrProcessing', error)
   return false 
}